### PR TITLE
Pretty up static pages

### PR DESCRIPTION
**Pull request by susannasouv**

### CODE REVIEW

**Comment by susannasouv** [comment]
Travis CI failures are due to "ambiguous matches" when clicking navbar items. Seems to be due to the fact that there is an exact copy of the menu items for the mobile view. Will discuss with team the best way to fix this.

**Comment by jeremyjpark** [comment]
Just to clarify, the review should start from e61a25d?

File: Gemfile [hunk 1]
```diff
@@ -48,3 +48,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# MaterializeCSS: Style choice
+gem 'materialize-sass'
```
Comment (jeremyjpark) [inline Gemfile]: This might trigger something in coeclimate's stylecheck

**Comment by susannasouv** [comment]
Not quite, review includes commit 2989fa6 but goes e61a25d and beyond.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -13,5 +13,23 @@
...
+    var width = $( window ).width();
+    if ( width > 992 ) {
+        $( 'ul#mobile-menu' ).removeClass('side-nav');
+        $( 'ul#mobile-menu' ).addClass('right hide-on-med-and-down');
+        $( 'ul#mobile-menu' ).css('transform', '');
+    } else {
+        $( 'ul#mobile-menu' ).addClass('side-nav');
+        $( 'ul#mobile-menu' ).removeClass('right hide-on-med-and-down');
+    }
+});
```
Comment (jeremyjpark) [inline app/assets/javascripts/application.js]: Same here (this one was my fault)

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -12,4 +12,18 @@
...
+@import 'materialize';
+@import "materialize/extras/nouislider";
+
+.nav-wrapper {
+  padding: 0em 1em 0em 1em;
+}
+
+.nav-wrapper .navbar-non-link {
+  padding: 0 15px;
+}
```
Comment (jeremyjpark) [inline app/assets/stylesheets/application.scss]: Another one

**Review by jeremyjpark — COMMENTED** [review]
Other than the newlines (of which one was my fault, LGTM.

**Comment by jeremyjpark** [comment]
:+1:

**Review by aganpule — APPROVED** [review]
yay this looks so much nicer!

**Pull request merged**